### DISCUSSION TITLE
How can I compile openvscode-server so I don't have to have node installed

### DISCUSSION THREAD

**Discussion by ubarilan** [gitpod-io/openvscode-server#412] [Q&A]
Hello,
how can I compile this project into an executable so that I can use it without having to have NodeJS installed on the machine that openvscode-server is running on?

**Comment by jeanp413** [comment]
@ubarilan  when you compile openvscode-server it's already bundled with node so you don't need to install separately

**Reply by ubarilan** [reply]
I understand that, but I would appreciate an executable a whole lot.
It can be much easier to install on new servers that way.
